cg/meta/workflow/mutant/mutant.py

```diff
@@ -148,49 +156,6 @@ def get_lims_naming_metadata(self, sample: Sample) -> str | None:
 
         return f"{region_code}_{lab_code}_{sample.name}"
 
-    def run_analysis(self, case_id: str, dry_run: bool, config_artic: str = None) -> None:
```

Comment:
From what I understand, this could has only be shifted down? I

t is usually a good idea to try to limit changes like this in a PR introducing new logic because it makes it more difficult to identify the actual changes and keep the diff as clean and minimal as possible.